### DISCUSSION TITLE
Ubuntu 16.04.7 LTS - Update (End of Standard Support for 14.04.6 04.2019)

### DISCUSSION THREAD

**Discussion by ghost** [freelawproject/courtlistener#1554] [Q&A]
Dear developers,
we tried to run courtlistener with Ubuntu 16.04.7 LTS but we did not get it working. Is out there experience on installing courtlistener on Ubuntu 16.04.* ?
Best regards and thanks for your work and support
Axel Arnold  Bangert
Wolfstrasse             2
52134 Herzogenrath

**Comment by mlissner** [comment]
Can you so what went wrong?

**Reply by ghost** [reply]
Hi Mr. Lissner,
there is a never ending list. But we stop trying that. We try to get the docker image running -> https://github.com/freelawproject/courtlistener/discussions/1556
Thank you very much for your help.
Axel

**Reply by mlissner** [reply]
Sounds good. I'm going to mark this as resolved. I guess if you want to file a bug or PR to fix the docker compose thing, I'd welcome that.
